README.md

```diff
@@ -72,6 +95,21 @@ GET example.com/api/v1/blogs/1
 GET example.com/api/v1/tech/blogs/1
 ```
 
+To sort your results, use **sort** as a query parameter. **sort** parameter is format of `?sort[sort_order]=sort_field`. If `sort_order` is not specified, the default sort order `asc` will be used
```

Comment:
This section of the documentation should only be in #11.

Reply:
removed